docs/src/reference-guide/components/graphql/queries.md

```diff
@@ -137,16 +140,7 @@ query {
 }
 ```
 
-Let's assume that we've created an indexer for this data and indexed data has been stored in the database. Now we'll send a request to the API server in the same way that we did for the basic query example:
-
-```txt
-curl -X POST http://localhost:29987/api/graph/fuel_examples/nested_query_index \
-   -H 'content-type: application/json' \
-   -d '{"query": "query { character { name book { title library { name city { name } } } } }", "params": "b"}' \
-| json_pp
-```
-
-And we receive the following response:
+Let's assume that we've created an indexer for this data and indexed data has been stored in the database. If we send the query, we'll get the following response:
```

Comment:
A missing 'the'

"... and "the" indexed data has been stored in the database."